modular_skyrat/modules/death_consequences_perk/death_consequences_trauma.dm

```diff
@@ -365,7 +365,8 @@
 /// The proc we call when we permanently kill our victim due to being at maximum degradation. DNRs them, ghosts/kills them, and prints a series of highly dramatic messages,
 /// befitting for a death such as this.
 /datum/brain_trauma/severe/death_consequences/proc/and_so_your_story_ends()
-	owner.ckey = null//	ADD_TRAIT(owner, TRAIT_DNR, TRAUMA_TRAIT) // you're gone bro - BUBBER EDIT - More DNR FA_ICON_XING
+	ADD_TRAIT(owner, TRAIT_DNR, TRAUMA_TRAIT) // you're gone bro
```

Comment:
removing an unnecessary edit since TRAIT_DNR has no effects asides from being used in the DNR examine and DNR hud

Reply:
dnr hud works based on ckey linkage not the trait check here

Reply:
i wrote the DNR hud, it depends on TRAIT_DNR, not ckey linkage, see lines 71-83 of skyrat's neutral.dm, where the DNR quirk is defined

you may mean the icon that appears AFTER you die and get DNRed. the DNR hud is a red exclamation mark that signifies that someone will become DNRed when they die